### ISSUE TITLE
Is an update coming for NVDA 2025.X?

### ISSUE THREAD

**Issue by britechguy** [grisov/checkGestures#14]
The title pretty much covers it.  I can't determine whether one is waiting in the wings or not.

**Comment by grisov** [comment]
> The title pretty much covers it. I can't determine whether one is waiting in the wings or not.

Dear Brian,

Thank you for your interest in this add-on.

I will be adapting it to the latest version of NVDA soon—hopefully this weekend.

**Comment by britechguy** [comment]
Thanks very much for the update.  I just asked on the NVDA Add-On Developer's group if there is a way, if someone happens to be working on a "next NVDA version compatible" version of an add-on in advance, that an end user can determine this is the case.

I know that every add-on developer simply cannot post messages on the NVDA Group about updates to their add-ons, but if there were some way for an interested user to check for what's "waiting in the wings" it would be handy to know.

If you happen to know if there is a way, please do let me know what that might be.

Again, I really appreciate your sending this status report more than I can say.

**Comment by grisov** [comment]
Brian, I apologize for the delay.

There are also several requests for additional features in the add-on - I haven't forgotten about them.

I can't promise that I will implement everything quickly, but ensuring compatibility is the top priority at the moment, and I will address it soon.

Of course, I will post an update about the **Check Gestures** in the nvda-addons group.

Thank you very much for your interest in this add-on!

**Comment by grisov** [comment]
> The title pretty much covers it. I can't determine whether one is waiting in the wings or not.

Dear Brian,

I have updated the following add-ons to compatibility with the latest version of NVDA.

Check Gestures:
- [download](https://github.com/grisov/checkGestures/releases/download/v1.0.7/checkGestures-1.0.7.nvda-addon)
- [update issue](https://github.com/nvaccess/addon-datastore/issues/5813)

Quick Dictionary:
- [download](https://github.com/grisov/quickDictionary/releases/download/v2.2.2/quickDictionary-2.2.2.nvda-addon)
- [update issue](https://github.com/nvaccess/addon-datastore/issues/5815)

Unmute:
- [download](https://github.com/grisov/Unmute/releases/download/v1.6.0/unmute-1.6.0.nvda-addon)
- [update issue](https://github.com/nvaccess/addon-datastore/issues/5817)

Volume Adjustment:
- [download](https://github.com/grisov/NVDA_Volume_Adjustment/releases/download/v1.4.0/volumeAdjustment-1.4.0.nvda-addon)
- [update issue](https://github.com/nvaccess/addon-datastore/issues/5818)

I wish you a good end of the week!

Olek
